containers/ecr-viewer/src/app/utils/evaluate/fhir-paths.ts

```diff
@@ -314,6 +320,30 @@ const _fhirPathMappings: { [K in FhirPathKeys]: Omit<FhirPath<K>, "name"> } = {
     type: "ValueX",
     path: "Bundle.entry.resource.where(resourceType = 'Patient').maritalStatus",
   },
+  patientNationality: {
+    type: "string",
+    path: "Bundle.entry.resource.where(resourceType = 'Observation').where(code.coding.where(code = '186034007' and system = 'http://snomed.info/sct')).valueCodeableConcept.coding.display",
+  },
+  patientCountryResidence: {
+    type: "string",
+    path: "Bundle.entry.resource.where(resourceType = 'Observation').where(code.coding.where(code = '77983-5' and system = 'http://loinc.org')).valueCodeableConcept.coding.display",
+  },
+  patientDisabilityStatus: {
+    type: "Observation",
+    path: "Bundle.entry.resource.where(resourceType = 'Observation').where(meta.profile = 'http://hl7.org/fhir/us/ecr/StructureDefinition/us-ph-disability-status')",
+  },
+  disabilityStatusQuestion: {
+    type: "string",
+    path: "code.coding.display",
+  },
+  disabilityStatusValue: {
+    type: "boolean",
+    path: "valueBoolean",
+  },
```

Comment:
```suggestion
  disabilityStatusValue: {
    type: "ValueX",
    path: "value",
  },
```

generally I prefer leaving things more generic than not on the viewer side as I don't trust that the input data will be as consistent as we hope - what's your thinking around this?